t/ingredients_processing.t

```diff
@@ -833,7 +833,7 @@ my @tests = (
 				    'text' => 'Guaven'
 				  },
 				  {
-				    'id' => 'en:rosehip',
+				    'id' => 'en:rose hip',
```

Comment:
Spaces get replaced with `-`, so the id is `rose-hip`

Reply:
i noticed *after* i'd already pressed commit :D